recipe/meta.yaml

```diff
@@ -37,19 +39,25 @@ test:
     - rope.refactor.importutils
```

Comment:
You can add `rope.contrib.autoimport` as an import as well: https://github.com/python-rope/rope/tree/1.7.0/rope/contrib/autoimport